usaspending_api/download/v2/download_column_historical_lookups.py

```diff
@@ -1207,6 +1211,9 @@
                 ("prime_award_amount", "broker_subaward__award_amount"),
                 ("prime_award_base_action_date", "broker_subaward__action_date"),
                 ("prime_award_base_action_date_fiscal_year", None),  # Annotation is used to create this column
+                ("prime_award_period_of_performance_start_date", None),
```

Comment:
Can you add the annotation comment to the few lines missing?

Reply:
sure thing